Update deploy-helm-charts.md (#1886)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/helm/cd-helm-category/deploy-helm-charts.md

* If the deployment fails on the very first execution, then Harness will apply the `--uninstall` flag itself. You can see this in the logs under `Wait For Steady State`.
* If you want to pass in some command flags when Harness performs the `--uninstall`, enter uninstall in **Manifest Details** and enter in the relevant command flags.

> #### Authenticating with Google Cloud with Helm OCI Connector

Reviewer: ```suggestion
#### Authenticate using Google Cloud with Helm OCI connector
```

>  In order to authenticate with GCP properly with the Helm OCI Connector, you will need to provide a username and password.

Reviewer: ```suggestion
 To connect Harness using GCP with the Helm OCI connector, you must provide a username and password.
```

>  To connect Harness using GCP with a Helm OCI connector, you must provide the username and password to your Google service account.

Reviewer: ```suggestion
 To configure authentication for GCP with a Helm OCI connector, you must provide the username and password to your Google service account.
```

>  **Username**: This will be a _json_key or _json_key_base64 (json_key_base64 is preferable to use and encode your google service account file to base64)

Reviewer: ```suggestion
 **Username**: A _json_key or _json_key_base64. We recommend that you use the json_key_base64 because you can encode your google service account file to base64.
```

>  **Password**: This will be Google service account file content

Reviewer: ```suggestion
 **Password**: Your Google service account file content.
```
